client/webserver/site/src/localized_html/en-US/bodybuilder.tmpl

```diff
@@ -5,10 +5,10 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   {{- /* The above 2 meta tags *must* come first in the head; any other head content must come *after* these tags */ -}}
-  <link rel="icon" href="/img/favicon.png?v=z8A71BD">
+  <link rel="icon" href="/img/favicon.png?v=h877Tgq">
   <meta name="description" content="Decred DEX Client Web Portal">
   <title>{{.Title}}</title>
-  <link href="/css/style.css?v=lDtQNP" rel="stylesheet">
+  <link href="/css/style.css?v=h877Tgq" rel="stylesheet">
```

Comment:
No changes to favicon or css afaict.  No complaints though if you wanna keep the js and css matching, but the favicon can stay.